### PR TITLE
Remove system.user as a configuration setting, and instead just use t…

### PR DESCRIPTION
**Pull request by bdmogal**
…he current user of that the Master is running as for bootstrapping.

Build: http://builds.cask.co/browse/CDAP-RUT113

### CODE REVIEW

File: cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java [hunk 1]
```diff
@@ -707,14 +707,14 @@
...
         "security.authorization.extension.config.";
-      public static final String SYSTEM_USER = "security.authorization.system.user";
       public static final String CACHE_ENABLED = "security.authorization.cache.enabled";
       public static final String CACHE_TTL_SECS = "security.authorization.cache.ttl.secs";
       public static final String CACHE_REFRESH_INTERVAL_SECS = "security.authorization.cache.refresh.interval.secs";
       public static final String ADMIN_USERS = "security.authorization.admin.users";
     }
 
     /**
+     *
```
Comment (chtyim) [inline cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java]: unneeded.

**Comment by bdmogal** [comment]
@chtyim addressed comments. Tested on a cluster. Build failing on a flaky test so running again. Please take another look.

**Comment by bdmogal** [comment]
Green: http://builds.cask.co/browse/CDAP-RUT113-2

**Comment by chtyim** [comment]
LGTM.

**Comment by bdmogal** [comment]
Thanks @chtyim. Merging.

**Pull request merged**